### PR TITLE
Docs: `ProductCard` documentation

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?
`ProductCard `documentation.

### CODE REVIEW

File: apps/docs/docs/reference/ui/molecules/ProductCard.mdx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  ProductCardImage,
+} from '@faststore/ui'
+```
+
+## Usage
+
+### Default
+
+```tsx live
+ <LocationProvider>
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/molecules/ProductCard.mdx:34]: Mari, not sure about the reason, but can you check this `Provider` usage.

image

File: apps/docs/docs/reference/ui/molecules/ProductCard.mdx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+- Write the product's title in two lines.
+- For the call-to-action button, use action verbs such as *Buy*, *Save*, and *Add to cart*.
+
+### Don'ts
+- Do not use the `ProductCard` for empty state scenarios. If the case, use the `Skeleton` component.
+- Do not use more than three badges (if necessary).
+
+## Accessibility
+- The `img` elements must have an `alt` attribute. 
+- For images containing text, the alt description must includes the image's text.
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/molecules/ProductCard.mdx:181]:
```suggestion
- For images containing text, the `alt` description must includes the image's text. 
```

File: apps/docs/docs/reference/ui/molecules/ProductCard.mdx (hunk 1 above)
Comment (Mariana-Caetano) [reply apps/docs/docs/reference/ui/molecules/ProductCard.mdx:34]: Thanks for pointing that out, Fanny! I'm going to check it here.

File: apps/docs/docs/reference/ui/molecules/ProductCard.mdx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+`data-product-card-actions`
+`data-store-price`
+`data-store-badge`
+`data-store-button`
+
+
+## Best practices
+
+### Do's
+- Use high-quality images.
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/molecules/ProductCard.mdx:171]: high-quality images also means a good image of the product that can attract more customers? 🤔

**Review by hellofanny — APPROVED** [review]
@Mariana-Caetano Looks great! 🤩  

Can you also take the chance and move the `Card` component also to the `Molecules` section on the docs? Thanks!

**Review by carolinamenezes — APPROVED** [review]

File: apps/docs/docs/reference/ui/molecules/Card.mdx [hunk 4]
```diff
@@ -80,4 +76,26 @@ All Card-related components support all attributes also supported by the `<div>`
...
+`data-store-card-actions`
+
+## Best practices
+
+### Do's
+
+- Use attractive images to grab shoppers' attention.
+- Keep the card content short.
+- Keep a standard visual style. Otherwise, your design may look off-putting.
+- Use transparent-backgrounded images.
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/molecules/Card.mdx:88]: Not sure about this one 🤔

File: apps/docs/docs/reference/ui/molecules/Card.mdx [hunk 5]
```diff
@@ -80,4 +76,26 @@ All Card-related components support all attributes also supported by the `<div>`
...
+- Use animation and visual hints to improve the user experience.
+- Use the most appropriate card size for each device type.
+- Make the entire card clickable to create larger touch zones.
+
+### Don'ts
+
+- Don't use the `Card` component for product cards. Instead, use the [`ProductCard` component](/reference/ui/molecules/ProductCard).
+- Avoid cluttering cards with too many buttons and content. 
+- Avoid inline links.
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/molecules/Card.mdx:97]: What would be an example of it? 👀

File: apps/docs/docs/reference/ui/molecules/Card.mdx [hunk 6]
```diff
@@ -80,4 +76,26 @@ All Card-related components support all attributes also supported by the `<div>`
...
+
+## Accessibility
+
+- Text elements and buttons must have sufficient color contrast against the background.
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/molecules/Card.mdx:101]: We can also add:
- For images containing text, the `alt` description must includes the image's text.

**Pull request merged**